app/app.ts

```diff
@@ -26,10 +25,18 @@ import VeeValidate from 'vee-validate';
 import ChildWindow from 'components/windows/ChildWindow.vue';
 
 const { ipcRenderer, remote } = electron;
-
 const slobsVersion = remote.process.env.SLOBS_VERSION;
 const isProduction = process.env.NODE_ENV === 'production';
 
+window['obs'] = window['require']('obs-studio-node');
+
+{ // Set up things for IPC
+  // Connect to the IPC Server
+  window['obs'].IPC.connect(process.env.SLOBS_IPC_PATH);
+  document.addEventListener('close', (e) => {
+    window['obs'].IPC.disconnect();
+  });
+}
```

Comment:
What's the purpose of this block here?

Reply:
I was more curious about why the unnamed block `{` `}`.  It isn't serving any purpose for the code.  Is this just meant to indicate these statements are a logical group?  I can't say I've ever seen anyone use unnamed blocks in javascript like this before.

Reply:
Yep, it's just to separate the logical execution of the code. I've recently started using this notation after learning rust (language) and at least for me it makes understanding the code much easier if i come back to it later. It serves no other purpose, so I can remove it if you want.